execution/gethexec/express_lane_service.go

```diff
@@ -368,9 +371,23 @@ func (es *expressLaneService) validateExpressLaneTx(msg *timeboost.ExpressLaneSu
 	if msg.AuctionContractAddress != es.auctionContractAddr {
 		return errors.Wrapf(timeboost.ErrWrongAuctionContract, "msg auction contract address %s does not match sequencer auction contract address %s", msg.AuctionContractAddress, es.auctionContractAddr)
 	}
-	currentRound := timeboost.CurrentRound(es.initialTimestamp, es.roundDuration)
-	if msg.Round != currentRound {
-		return errors.Wrapf(timeboost.ErrBadRoundNumber, "express lane tx round %d does not match current round %d", msg.Round, currentRound)
+
+	currentTime := time.Now()
+	for {
+		currentRound := timeboost.CurrentRound(es.initialTimestamp, es.roundDuration)
+		if msg.Round == currentRound {
+			break
+		}
+
+		if msg.Round == currentRound+1 &&
+			timeboost.TimeTilNextRoundAfterTimestamp(es.initialTimestamp, currentTime, es.roundDuration) <= es.earlySubmissionGrace {
```

Comment:
currentTime must be updated inside the loop so if we're missing a few ms we'll only sleep for a few ms.

Reply:
Fixed this.